nf_core/pipeline-template/README.md

```diff
@@ -25,57 +25,76 @@
 
 ## Introduction
 
-<!-- TODO nf-core: Write a 1-2 sentence summary of what data the pipeline is for and what it does -->
+<!-- TODO nf-core: Write a 2-3 sentence summary of what data the pipeline is for and what it does, e.g.
 
-**{{ name }}** is a bioinformatics best-practice analysis pipeline for {{ description }}.
+   > **nf-core/rnaseq** is a bioinformatics pipeline that can be used to analyse RNA sequencing data obtained from
```

Comment:
```suggestion
   > **{{name}}** is a bioinformatics pipeline that can be used to analyse RNA sequencing data obtained from
```

Reply:
I think we should avoid anything about the rnaseq pipeline, rather keep this generic.